cypress/integration/dresses-list.spec.ts

```diff
@@ -0,0 +1,26 @@
+import {MenuContentPage, DressesListPage} from "../page/index";
+
+describe("the user navigates to the dresses page should", () => {
+  let menuContentPage: MenuContentPage;
+  let dressesListPage: DressesListPage;
+
+  before(() => {
+    menuContentPage = new MenuContentPage();
+    dressesListPage = new DressesListPage();
+  });
+
+  it("show the available dresses", () => {
+    // arrange
+    const allDresses =
+    ["Printed Dress", "Printed Dress", "Printed Summer Dress", "Printed Summer Dress", "Printed Chiffon Dress"];
```

Comment:
I suggest that this const array can be part of the before with the initialization of the pages, i think its better placement as they are expected elements. Difference for the user and passwords as they are variables that we require for the execution of the test